examples/CMakeLists.txt

```diff
@@ -7,5 +7,6 @@ macro(add_example demo_name)
 endmacro(add_example target_name)
 
 
-add_example(leg_example)
-add_example(spin_motor_example)
\ No newline at end of file
+# add_example(leg_example)
```

Comment:
Please fix for final version. Likely your new example should just be merged into spin motor